src/Filter/EmptyFilter.php

```diff
@@ -13,70 +13,16 @@
 
 namespace Sonata\DoctrineORMAdminBundle\Filter;
 
-use Sonata\AdminBundle\Datagrid\ProxyQueryInterface as BaseProxyQueryInterface;
-use Sonata\AdminBundle\Form\Type\Filter\DefaultType;
-use Sonata\DoctrineORMAdminBundle\Datagrid\ProxyQueryInterface;
-use Sonata\Form\Type\BooleanType;
-use Symfony\Component\Form\Extension\Core\Type\HiddenType;
-
-final class EmptyFilter extends Filter
+// NEXT_MAJOR: remove this file
+@trigger_error(sprintf(
```

Comment:
I guess this will trigger an unavoidable deprecation

Reply:
How would you do instead ?

Reply:
Good catch @franmomu !

 You can trigger a deprecation if you try to construct a new instance.

Reply:
Fixed here: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1236